zap/src/main/java/org/zaproxy/zap/extension/httppanel/view/syntaxhighlight/lexers/WwwFormTokenMaker.java

```diff
@@ -172,7 +172,7 @@ private static int zzUnpackTrans(String packed, int offset, int [] result) {
 
   /* error messages for the codes above */
   private static final String[] ZZ_ERROR_MSG = {
-    "Unkown internal scanner error",
+    "Unknown internal scanner error",
```

Comment:
This is generated code.

Reply:
If you can point me to the repository that is generating this, I'll try to send it fixes at some point...

Reply:
It seems it was already fixed: jflex-de/jflex@be361f6e13b8913b7976b6b0ff8a9d65a80a2978

Reply:
Thanks. I've dropped it. If someone wants to rebuild the jflex output, they can do that in their own PR.

Reply:
Sounds good.